### DISCUSSION TITLE
请问conformer_wenetspeech预训练模型融合语言模型如何实现

### DISCUSSION THREAD

**Discussion by Will-2017** [PaddlePaddle/PaddleSpeech#2293] [Q&A] [answered]
我在做ASR的时候，比较了deepspeech2online_wenetspeech和conformer_wenetspeech两个模型，发现后者效果比前者好很多，然后我准备了领域相关的语料训练了一个个性化的语言模型，通过修改ds2的模型配置中的lang_model_path成功与ds2集成效果提升不少。但是在看conformer代码的时候没看到这样的配置，有大神知道该怎么实现吗？

**Comment by Zth9730** [comment, marked as answer]
PaddleSpeech支持transformer decoder联合LM解码
参考[PaddleSpeech/tree/develop/examples/librispeech/asr2](https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/examples/librispeech/asr2)
